Makefile

```diff
@@ -51,15 +51,15 @@ check: nqueen $(TESTS)
 	tests/driver.sh
 
 tests/%.s: tests/%.c $(TARGET)
-	./mzcc < $< > $@
+	./mzcc -o $@ < $< - 
```

Comment:
Don't abuse `-`. Keep the command line as similar as gcc does.

Reply:
Ohh...I was out of my mind Orz